src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Offender.java

```diff
@@ -96,31 +94,27 @@
     private List<OffenderIdentifier> identifiers = new ArrayList<>();
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @NotFound(action = IGNORE)
```

Comment:
Does removing this indicate it is impossible for this not to be found?

Reply:
It means that if there is a value in there that is not matched against a FK it would cope but gender and ethnicity should always be a valid value